classes/socket.h

```diff
@@ -111,18 +114,23 @@ ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(Socket_getHost, 0, 0, IS_ARRAY, 0)
 	ZEND_ARG_TYPE_INFO(0, port, _IS_BOOL, 0)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(Socket_select, 0, 3, IS_LONG, 0)
+ZEND_BEGIN_ARG_INFO_EX(Socket_select, 0, 0, 4)
```

Comment:
In the code the `sec` parameter still appears to be optional, so the required argcount should be 3.